src/solvers/dgmulti/dg.jl

```diff
@@ -201,7 +201,7 @@ end
 
 # for the stepsize callback
 function max_dt(u, t, mesh::DGMultiMesh,
-                constant_speed::False, equations, dg::DGMulti{NDIMS}, cache) where {NDIMS}
+                constant_speed::False, equations, dg::DGMulti{NDIMS}, polydeg::Int, cache) where {NDIMS}
```

Comment:
```suggestion
                constant_speed::False, equations, dg::DGMulti{NDIMS}, polydeg::Integer, cache) where {NDIMS}
```
We usually dispatch on `Integer`, not on `Int` specifically.